package/MDAnalysis/core/accessors.py

```diff
@@ -63,10 +63,14 @@
    :members:
 
 """
+from __future__ import absolute_import
+from __future__ import annotations
 
 from functools import partial, update_wrapper
 
 from .. import _CONVERTERS
+import typing as t
```

Comment:
minor, I think: I often see objects from `typing` imported directly, i.e., `from typing import Optional, Dict`. I guess this may be personal preference though.

Reply:
Ill make the changes.